app/templates/index.html

```diff
@@ -52,7 +52,7 @@
       {% set items = items + [
         {
           "link": "/g-cloud",
-          "title": "Find cloud hosting, software and support" if show_new_g9_content else "Find cloud technology and support",
+          "title": "Find " + gcloud_framework_description,
```

Comment:
Everywhere else I've worked for a long time, this would be a no-no because translations. I suppose in this case it's fine, but having it stored as a separate string would have been fine too :)